injector/LoggerInstance/AdliLogger.py

```
@@ -3,6 +3,8 @@
 from clp_logging.handlers import ClpKeyValuePairStreamHandler
 
 import traceback
+import threading
+import asyncio
```

Ruff (0.11.9)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)